### PR TITLE
Add age personal stat and fix points bug

### PR DESCRIPTION
**Pull request by Natty-Boh**

### CODE REVIEW

File: src/handler/global/personalstats.ts [hunk 1]
```diff
@@ -147,6 +149,9 @@ function getPoints(user: PrivateUserData): number {
         const n = user.skills[skill as keyof Skills];
         points += (n * (n + 1)) / 2;
     }
+    if (user.unlockedUnits?.length !== 0) {
```
Comment (TotallyNot) [inline src/handler/global/personalstats.ts:152]: linter warning here

Comment (Natty-Boh) [reply src/handler/global/personalstats.ts:152]: for some reason running the linter isnt working locallym and its passing the build here... did you break it?

Comment (TotallyNot) [reply src/handler/global/personalstats.ts:152]: works just fine for me

Comment (TotallyNot) [reply src/handler/global/personalstats.ts:152]: just remove the unnecessary optional chaining >.>

**Review by TotallyNot — APPROVED** [review]

**Pull request merged**